float64[] tcp # [X, Y, Z, RX, RY, RZ]
float32 weight
---